apps/mudita-center-e2e/src/specs/stress-tests/connected-devices-stress-test.ts

```diff
@@ -0,0 +1,334 @@
+  const firstSerialNumber = "KOM1234567890"
+  const secondSerialNumber = "KOM1234567892"
+  const thirdSerialNumber = "KOM1234567893"
+  const fourthSerialNumber = "KOM1234567894"
+  const fifthSerialNumber = "KOM1234567895"
+  const sixthSerialNumber = "KOM1234567896"
+
+  before(async () => {
+    E2EMockClient.connect()
+    await browser.waitUntil(() => {
+      return E2EMockClient.checkConnection()
+    })
+  })
+
+  after(() => {
+    E2EMockClient.stopServer()
+    E2EMockClient.disconnect()
+  })
+
+  it("Connect 1,2 Kompakt devices", async () => {
+    const devices = [
+      {
+        path: "path-1",
+        body: overviewDataWithOneSimCard,
+        serialNumber: firstSerialNumber,
+      },
+      {
+        path: "path-2",
+        body: overviewDataWithOneSimCard2nd,
+        serialNumber: secondSerialNumber,
+      },
+    ]
+
+    for (const device of devices) {
+      E2EMockClient.mockResponse({
+        path: device.path,
+        body: device.body,
+        endpoint: "FEATURE_DATA",
+        method: "GET",
+        status: 200,
+      })
+      E2EMockClient.addDevice({
+        path: device.path,
+        serialNumber: device.serialNumber,
+      })
+
+      await browser.pause(6000)
+    }
+  })
+
+  it("Check Drawer Modal and Switch to 1st device", async () => {
+    const deviceSelectDrawer = await drawerPage.deviceSelectDrawer
+    await expect(deviceSelectDrawer).toBeDisplayed()
+
+    const deviceImageOnDrawer = await drawerPage.deviceImageOnDrawer
+    await expect(deviceImageOnDrawer).toBeDisplayed()
+
+    const firstDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      firstSerialNumber
+    )
+    const secondDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      secondSerialNumber
+    )
+    await expect(firstDeviceOnDrawer).toBeDisplayed()
+    await firstDeviceOnDrawer.waitForClickable()
+    await firstDeviceOnDrawer.click()
+  })
+  it("Verify Overview Page for 1st device", async () => {
+    const kompaktImage = await OverviewKompaktPage.kompaktImage
+    await expect(kompaktImage).toBeDisplayed()
+    await expect(kompaktImage).toHaveAttribute("src", kompaktImageRegex)
+
+    const kompaktOsVersion = await OverviewKompaktPage.kompaktOsVersion
+    await expect(kompaktOsVersion).toBeDisplayed()
+
+    const kompaktSimCard1Subtext =
+      await OverviewKompaktPage.kompaktSimCard1Subtext
+    await expect(kompaktSimCard1Subtext).toHaveText("SIM 1")
+
+    const kompaktNetworkName = await OverviewKompaktPage.kompaktNetworkName
+    await expect(kompaktNetworkName).toBeDisplayed()
+    await expect(kompaktNetworkName).toHaveText("T-Mobile")
+
+    const kompaktBatteryLevelValue =
+      await OverviewKompaktPage.kompaktBatteryLevelValue
+    await expect(kompaktBatteryLevelValue).toBeDisplayed()
+    await expect(kompaktBatteryLevelValue).toHaveText("100%")
+  })
+
+  it("Switch to 2nd device", async () => {
+    const secondDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
```

Comment:
I don't see the part where the drawer is opened. Am I missing something?

Reply:
it is opened while 2nd device is connected, then it clicks on the 2nd device from the top and opens it for further verification of the elements on the overview page